cpp/ql/src/Likely Bugs/OO/UnsafeUseOfThis.qhelp

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+
+<overview>
+<p>This rule finds calls to pure virtual member functions in constructors and destructors. Such calls do not perform virtual dispatch, and can cause undefined behavior.</p>
+
+</overview>
+<recommendation>
+<p>Do not rely on virtual dispatch in constructors and destructors. Instead, each class should be responsible for acquiring and releasing its resources.</p>
```

Comment:
Can we add a more actionable recommendation for when the idealistic solution can't work? For example, recommend the [Dynamic Binding During Initialization idiom](https://isocpp.org/wiki/faq/strange-inheritance#calling-virtuals-from-ctor-idiom).

Reply:
Done in 4a7f9100e41d58cb7f8a96784307ce40fd2a4aee.